doc/userguide/support-status.rst

```diff
@@ -0,0 +1,490 @@
+   +------------------+----------+----------+-------+--------------------------------+
+   |OpenBSD           |6.9       |Core team |       |                                |
+   +------------------+----------+----------+-------+--------------------------------+
+   |OSX/macOS         |??        |Core team |       |                                |
+   +------------------+----------+----------+-------+--------------------------------+
+   |Windows/MinGW64   |Windows 10|Core team |       |5.0+                            |
+   +------------------+----------+----------+-------+--------------------------------+
+
+Community
+~~~~~~~~~
+
+.. table::
+   :widths: 20 15 15 10 40
+   :width: 100%
+
+   +------------------+----------+----------+-------+--------------------------------+
+   |Architecture      |Version   |Support   |QA     | Notes                          |
+   +==================+==========+==========+=======+================================+
+   |Windows/MinGW32   |Windows 10|Core team |       |5.0+                            |
+   +------------------+----------+----------+-------+--------------------------------+
+ 
+Architecture Support
+--------------------
+
+Tier 1
+~~~~~~
+
+.. table::
+   :widths: 15 15 30 40
+   :width: 100%
+
+   +-------------+-------------+-------------+-------------+
+   |Architecture |Support      |QA           |Notes        |
+   +=============+=============+=============+=============+
+   |x86_64       |Core team    |             |             |
+   +-------------+-------------+-------------+-------------+
+   |ARM8-64bit   |Core team    |             |             |
+   +-------------+-------------+-------------+-------------+
+
+Tier 2
+~~~~~~
+
+.. table::
+   :widths: 15 15 30 40
+   :width: 100%
+
+   +-------------+-------------+-------------+-------------+
+   |Architecture |Support      |QA           |Notes        |
+   +=============+=============+=============+=============+
+   |ARM7-32bit   |Core team    |             |             |
+   +-------------+-------------+-------------+-------------+
+   |i386         |Core team    |             |             |
+   +-------------+-------------+-------------+-------------+
+
+Community
+~~~~~~~~~
+
+.. table::
+   :widths: 15 15 30 40
+   :width: 100%
+
+   +-------------+-------------+---------------------------+---------------------------------------------+
+   |Architecture |Support      |QA                         |Notes                                        |
+   +=============+=============+===========================+=============================================+
+   |PPC64el      |             |Part of Fedora automated QA|Access can be arranged through IBM dev cloud |
+   +-------------+-------------+---------------------------+---------------------------------------------+
+   |PPC64        |Victor Julien|                           |No access except an old Mac G5 Victor has    |
```

Comment:
can remove references to me here, that hw is no longer functional

should add riscv here